docs/UserGuide.md

```diff
@@ -22,29 +22,35 @@ dedicated platform catered to helping them manage their clientele. This is where
 ## Table of Contents
 
 - [Quick start](#quick-start)
+- [User Interface Layout](#user-interface-layout)
```

Comment:
think it would be could if we could split up the features into seperate sections instead of having one long list
eg;
1. Client features
    - ...
 2. Policy features
    - ...